substrate/frame/executive/src/lib.rs

```diff
@@ -487,6 +487,12 @@ where
 		let mut weight = Weight::zero();
 		if Self::runtime_upgraded() {
 			weight = weight.saturating_add(Self::execute_on_runtime_upgrade());
+
+			frame_system::LastRuntimeUpgrade::<System>::put(
+				frame_system::LastRuntimeUpgradeInfo::from(
+					<System::Version as frame_support::traits::Get<_>>::get(),
+				),
+			);
```

Comment:
How was this value being set prior to this PR?

Reply:
In the `runtime_upgraded`.